src/scenes/Projects/queries.graphql

```diff
@@ -0,0 +1,68 @@
+query projects($input: ProjectListInput!) {
+  projects(input: $input) {
+    ...projects
+  }
+}
+
+fragment projects on ProjectListOutput {
+  items {
```

Comment:
What's really needed here is a fragment for the item in the list. We will only fetch a limited number of properties for each project when listing to keep the data down. 

```graphql
fragment ProjectListItem on Project {
  ...
}
```
The interface generated from this, `ProjectListItemFragment` can be used on the `ProjectListItemCard` instead of doing a `Partial<Project>`. The former accurately reflects the properties available, where the latter is a total guess.


With that fragment in place the query is only a couple lines making a fragment not really worth it.
```graphql
projects(input: $input) {
  items {
    ...ProjectListItem
  }
  total
  hasMore
}
```